Clinical-Genomics/scout
# Minified splice junction tracks that get saved on case load

Fix #2541

This PR introduces minified (contain only genes loaded in Scout demo) splice junction tracks for the affected individual of the demo file. So the demo case will always have an "RNA splicing" button to visualize this type of data for its variants.

**How to test**:
1. Try loading demo case with the command: `scout setup demo`
2. Try loading the demo case with the command scout --demo load case `scout --demo load case scout/demo/643594.config.yaml -u`
3. Make sure the splice junction tracks are visible for case variants.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
